frontmatter/what-you-need.markdown

```diff
@@ -13,3 +26,11 @@ To follow along with this book, you'll need the following:
 
 * **Technology Two**: Fill out what the reader needs; a certain OS version, a certain IDE version, etc.
 
+# Example
```

Comment:
I don't think an example is needed here. It is quite obvious from the above section what is needed, I don't think an example will provide anymore clarity.

Reply:
Hey Edith!

You'd think that, but then you take a look at this book for example: https://www.raywenderlich.com/books/dagger-by-tutorials/v1.0/chapters/i-what-you-need and there is no language and no Android or platform version, or Dagger/Hilt versions included.

One of the reasons we have all these instructions is that even some things which are basic logic are often not filled. :shrug:

Reply:
Yes, but leaving an in-dept example doesn't mean that people will add any more information. Could be that these were not important to the author's a the time of writing, and the minimum needed is just what they added there? or maybe it is addressed somehow in the book that makes it obvious not to add here. 
What I am saying is, you've expanded the "what you need" section already by adding lines for what we'd expect like the Framework and OS versioning which already have samples in them, I just feel that adding a specific example is too much, and not necessarily helpful.